src/main/java/uk/gov/hmcts/divorce/ciccase/model/ApplicantCIC.java

```diff
@@ -8,6 +8,7 @@
 @Getter
 @AllArgsConstructor
 public enum ApplicantCIC implements HasLabel {
+
     @JsonProperty("ApplicantCIC")
     APPLICANT_CIC("Applicant (If different from subject).");
 
```

Comment:
Minor one: can we make `i` lower case in `If` and remove full stop at the end to match with the text in the requirement screen shot?